### DISCUSSION TITLE
How do I create a secondary app?

### DISCUSSION THREAD

**Discussion by nandorojo** [invertase/react-native-firebase#6029] [Q&A]
Quick question re the docs: https://rnfirebase.io/app/usage#initializing-secondary-apps

It looks like the config object for the secondary app matches the one from Firebase JS. Does this mean I should create a new `web` app on Firebase's console? Or should I create an iOS project, look through the Plist file, and copy over the config variables? Thanks!

**Comment by nandorojo** [comment]
Also, can my secondary app be from a different firebase account?

**Reply by mikehardy** [reply]
Hmm. I haven't tried that. But I don't see why not? Should work? Interested to hear that it works or not though

**Reply by nandorojo** [reply]
i'll let you know. do you know the answer to the first question btw?

**Comment by mikehardy** [comment]
> Quick question re the docs: https://rnfirebase.io/app/usage#initializing-secondary-apps
> 
> It looks like the config object for the secondary app matches the one from Firebase JS. Does this mean I should create a new `web` app on Firebase's console? Or should I create an iOS project, look through the Plist file, and copy over the config variables? Thanks!

I think you will want your secondary apps to match the correct app type, otherwise things will look pretty strange in the console
